Ined.js

```diff
@@ -0,0 +1,550 @@
+{
+	"translatorID": "dd5761aa-4145-4911-b45c-16c78cfc24c8",
+	"label": "Ined",
+	"creator": "Mysciencework",
+	"target": "^https?://archined(-preprod)?\\.ined\\.fr",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2022-05-23 18:02:42"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+	Mysciencework translator
+	Copyright © Mysciencework
+	
+	This file is part of Zotero.
+	
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+	
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU Affero General Public License for more details.
+	
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero.  If not, see <http://www.gnu.org/licenses/>.
+	
+	***** END LICENSE BLOCK *****
+*/
+function detectWeb(doc, url) {
+	function findItemType(doc, url) {
+
+		var risUrl = url.replace(/(\/view\/)/, "/api/public/v2/view/") + "/ris";
+		var response;
+		const httpRequest = new XMLHttpRequest();

```

Comment:
As the site is a single page application (SPA), we do not get the content of the page in the input parameter doc. This is why we are forced to make the request.

Reply:
You can update the available DOM using `Z.monitorDOMChanges` as used e.g. here: https://github.com/zotero/translators/blob/1d1ce6111ac17a4064c1d8836f6beec54404c32b/Microsoft%20Academic.js#L47

Since I understand this is your site, you can also force Zotero to re-trigger from your site code, see https://www.zotero.org/support/dev/exposing_metadata#force_zotero_to_refresh_metadata

Reply:
With `defer: true` I was able to test and get the content of the page, thanks for the tip. This has allow us to remove the remote calls from `detectWeb()`

Is there a way to test that `monitorDOMChanges()` actually does the expected? (basically that we got right the element to monitor)